openapi-public.yaml

```
@@ -1508,6 +1518,40 @@ components:
           description: "The ID of the newly created email address"
           allOf:
             - $ref: '#/components/schemas/UUID4'
+    JWTClaims:
+      type: object
+      description: "The claims extracted from a JWT."
+      properties:
+        subject:
+          description: "The unique identifier of the token's subject."
+          allOf:
+            - $ref: '#/components/schemas/UUID4'
+        issued_at:
+          description: "The timestamp indicating when the token was issued."
+          type: string
+          format: date-time
+        expiration:
+          description: "The timestamp indicating when the token will expire."
+          type: string
+          format: date-time
+        audience:
+          description: "The intended audience of the token."
+          type: string
```

This is currently an array of strings, so:

```suggestion
          type: array
          items:
            type: string
```